### ISSUE TITLE
Can I join the mobile app?

### ISSUE THREAD

**Issue by lmuiotctf** [ioanmo226/chatgpt-web-application#7]
Hello author, you can adapt to the mobile phone code

**Comment by ioanmo226** [comment]
You mean you want me to develop mobile(ios, android) app too?

**Comment by lmuiotctf** [comment]
Yes, at present, Android open web page is not adapted, the page is freely scalable, can it be adapted to the Android iOS client, attach Android open website page, page size can be adapted.
![0d1307087e7999e38a3c1efb9054092](https://user-images.githubusercontent.com/52068443/218406442-a8445355-a95a-4b89-9679-5dfc581c6b7b.jpg)

**Comment by ioanmo226** [comment]
You mean you want me to make website mobile responsive or make native iOS/Android app?

**Comment by lmuiotctf** [comment]
Yes, the corresponding Android and iOS applications are adapted to the size of the mobile terminal

**Comment by ioanmo226** [comment]
Now site is mobile-responsive.
Please check and let me know.
![image](https://user-images.githubusercontent.com/77344191/218504794-693c6d59-36e9-4763-95bb-2780114c1ac8.png)
